### PR TITLE
Cleanup scritp enhancement

### PR DESCRIPTION
**Pull request by qpdpQ**
1. Allow user to input multiple namespaces
2. update common-service-maps after a tenant is uninstalled
3. when there is no `map-to-common-service-namespace` in the common-service-maps , cleanup control ns

### CODE REVIEW

**Comment by Daniel-Fan** [comment]
@qpdpQ 
When we delete `cs-control` namespace, we are missing to delete NamespaceScope CR `common-service`. Here is the [template](https://github.com/IBM/ibm-common-service-operator/blob/7ba99773c8cc7a3f16b3d2b61af3cee442005a85/controllers/constant/namespace.go#L109)
```
function cleanup_NamespaceScope() {
    local namespace=$1
    ${OC} delete namespacescope odlm-scope-managedby-odlm nss-odlm-scope nss-managedby-odlm -n ${namespace} --ignore-not-found
}
```

File: cp3pt0-deployment/uninstall_tenant.sh [hunk 1]
```diff
@@ -265,10 +300,84 @@ function delete_tenant_ns() {
...
+data:
+  common-service-maps.yaml: |
+${yaml}
+EOF
+)"
+    echo "$object" | oc apply -f -
+}
+
+# check if we need to cleanup contorl namespace and clean it
+function cleanup_cs_contorl() {
```
Comment (bluzarraga) [inline cp3pt0-deployment/uninstall_tenant.sh:352]: nitpick but a typo in the function name

**Review by bluzarraga — COMMENTED** [review]
When deleting one tenant from my cluster (where there were two), the script cleaned up my cs-control namespace as well. @qpdpQ and I have slacked about it but will add to this pr too

**Comment by bluzarraga** [comment]
Tested on amd64, successfully uninstalled multiple tenants, cleaned up control ns when it should have and left it alone when a tenant still existed, deletes csmaps when no more tenants are present. I think this looks good, thanks @qpdpQ

**Pull request merged**